api/middleware/middleware.go

```diff
@@ -37,6 +37,9 @@ func getOrg(req *http.Request, multiTenant bool) (int, error) {
 	}
 	orgStr := req.Header.Get("x-org-id")
 	if orgStr == "" {
+		orgStr = req.Header.Get("x-grafana-org-id")
+	}
+	if orgStr == "" {
```

Comment:
I am ok with this.  We can also pass this through graphite-web if you like, just need to update the "REMOTE_STORE_FORWARD_HEADERS in local_settings.py

https://github.com/raintank/deployment_tools/blob/master/metrictank/image/graphite-web/local_settings.py#L135